### DISCUSSION TITLE
Slatebox.com runs on kube-hetzner

### DISCUSSION THREAD

**Discussion by TimHeckel** [mysticaltech/terraform-hcloud-kube-hetzner#793] [Show and tell]
I just wanted to mention how grateful I am for this project, and how it has helped me affordably rollout my app at [slatebox.com](https://slatebox.com). The recent upgrade to the 2+ version included an autoscaler, which has been so great at elastically scaling pods based on traffic. I will put it all to the test this Tuesday, when [Slatebox launches](https://www.producthunt.com/products/slatebox). Please consider signing up to be notified - and also check back that day to see if my app has crashed or if it can keep up! 

Again, thanks so much -- huge props to @mysticaltech for all your support.

Tim

**Comment by mysticaltech** [comment]
Ah very good to see production use, and very nice to read those words! Congrats Tim, I will definitely sign up. @kube-hetzner/core FYI.
